documentation/index.md

```diff
@@ -0,0 +1,66 @@
+The Charmed Temporal K8s Operator delivers automated operations management from
+day 0 to day 2 on [Temporal](https://temporal.io/). It is an open source,
+end-to-end, production-ready workflow engine on top of [Juju](https://juju.is/).
+
+Temporal is a developer-first, open source platform that ensures the successful
+execution of services and applications (using workflows). Use Workflow as Code
+(TM) to build and operate resilient applications. Leverage developer friendly
+primitives and avoid fighting your infrastructure
+
+This operator provides a Temporal server, and consists of Python scripts which
+wraps the versions distributed by
+[temporalio](https://hub.docker.com/r/temporalio/server).
+
+The Charmed Temporal K8s operator offers features such as replication,
+observability and easy to use integration with applications. It addresses the
+requirement for deploying Temporal in a structured and uniform fashion, while
+allowing the user flexibility in configuration. It streamlines the process of
+deploying, scaling, configuring and overseeing Temporal at scale in a dependable
+manner for production purposes.
+
+## Project and community
+
+Temporal Server Charm is a member of the Ubuntu family. It’s an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+
+- [Read our Code of Conduct](https://ubuntu.com/community/code-of-conduct)
+- [Join the Discourse forum](https://discourse.charmhub.io/tag/temporal)
+- [Contribute and report bugs](https://github.com/canonical/temporal-k8s-operator)
+
+## In this documentation
+
+|                                                                                                                                                  |                                                                                               |
+| ------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------- |
+| [Tutorial](./tutorial/00-introduction.md) </br> Get started - a hands-on introduction to using Charmed Temporal K8s operator for new users </br> | [How-to guides](./how-to/) </br> Step-by-step guides covering key operations and common tasks |
+| [Reference](https://charmhub.io/temporal-k8s/actions) </br> Technical information - specifications, APIs, architecture                           | [Explanation](./explanation/) </br> Concepts - discussion and clarification of key topics     |
+
+# Navigation
+
+| Level | Path                | Navlink                                                               |
+| ----- | ------------------- | --------------------------------------------------------------------- |
+| 1     | tutorial            | [Tutorial]()                                                          |
```

Comment:
Are you leaving these empty on purpose?

Reply:
Yes, when these pages are created in Discourse, they will show up as menu headers on the charm's Charmhub page (an example [here](https://charmhub.io/postgresql-k8s)).